appinfo.json

```diff
@@ -6,7 +6,7 @@
   "versionCode": 4,
   "versionLabel": "1.3",
   "sdkVersion": "3",
-  "targetPlatforms": ["basalt"],
+  "targetPlatforms": ["basalt", "chalk"],
```

Comment:
The Basalt and Chalk UI guidelines do not apply to Aplite yet. Perhaps after 3.x for Aplite is a thing.